{"uri":"file:///Users/vega/Code/huli/manager-web/src/vendor/aws/aws-sdk-php/src/Aws/DataPipeline/Exception/InternalServiceErrorException.php","root":{"kind":0,"name":"","children":[{"kind":512,"name":"Aws\\DataPipeline\\Exception","location":[16,0,16,37]},{"kind":1,"name":"Aws\\DataPipeline\\Exception\\InternalServiceErrorException","location":[21,0,21,68],"associated":[{"kind":1,"name":"Aws\\DataPipeline\\Exception\\DataPipelineException"}],"description":"An internal service error occurred."}]}}